app/upgrades.go

```diff
@@ -387,21 +389,16 @@ func migrateMarkerParams(ctx sdk.Context, app *App) {
 // TODO: Remove with the umber handlers.
 func migrateMetadataOSLocatorParams(ctx sdk.Context, app *App) {
 	ctx.Logger().Info("Migrating metadata os locator params.")
-	metadataParamSpace := app.ParamsKeeper.Subspace(metadatatypes.ModuleName).WithKeyTable(metadatatypes.ParamKeyTable())
-	maxValueLength := uint32(metadatatypes.DefaultMaxURILength)
-	// TODO: remove metadatatypes.ParamStoreKeyMaxValueLength with the umber handlers.
-	if metadataParamSpace.Has(ctx, metadatatypes.ParamStoreKeyMaxValueLength) {
-		metadataParamSpace.Get(ctx, metadatatypes.ParamStoreKeyMaxValueLength, &maxValueLength)
-	}
-	app.MetadataKeeper.SetOSLocatorParams(ctx, metadatatypes.OSLocatorParams{MaxUriLength: maxValueLength})
+	params := metadatatypes.DefaultOSLocatorParams()
+	app.MetadataKeeper.SetOSLocatorParams(ctx, params)
 	ctx.Logger().Info("Done migrating metadata os locator params.")
 }
```

Comment:
As far as I can tell, the os-locator params never actually got set up in the params module. When I try to query for them, it says that mainnet has `"0"`, and testnet has `""`. So I updated this step to just set it to the default which is what was always being used anyway.